src/lib/proxy/openaiFormat.ts

```
@@ -0,0 +1,959 @@
+        // ping, error, and unknown events are ignored.
+        return;
+    }
+  };
+
+  return new TransformStream<Uint8Array, Uint8Array>({
+    transform(chunk, controller) {
+      buffer += decoder.decode(chunk, { stream: true });
+
+      // Claude SSE events are separated by blank lines (`\n\n`).
+      let sepIdx: number;
+      while ((sepIdx = buffer.indexOf("\n\n")) !== -1) {
```

Biome (2.4.15)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)